console/templates/partials/navigation_tabs.html

```diff
@@ -1,32 +1,65 @@
-<ul class="navigation-tabs navigation-tabs--component">
 
+<ul class="navigation-tabs navigation-tabs--component">
+  {% if request.path == '/submit' %}
+    <li id="Submit-active"
+        class="navigation-tabs__item navigation-tabs__item--primary">
+        <a href="/submit"
+           class="btn btn--secondary navigation-tabs__tab"> Submit </a>
+    </li>
+  {% else %}
     <li id="Submit"
-        class="navigation-tabs__item navigation-tabs__item--primary {% if request.path == '/submit' %} navigation-tabs__item--active {% endif %}">
+        class="navigation-tabs__item navigation-tabs__item--primary">
```

Comment:
These blocks look very similar is possible to put the if statement around the bit that actually changes? I.e. the additional class?